packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -56,7 +56,7 @@ export const useRootStyles = makeStyles<BadgeState>([
       gap: '4px',
       paddingRight: '8px',
       paddingLeft: '8px',
-      fontSize: '12px',
+      fontSize: '10px',
```

Comment:
I forget if I mentioned this in the `Badge` review, but I think these should be using font tokens instead, like `theme.global.type.fontSizes.base[200]`

Reply:
Actually it's using some tokens like `Caption 2` in the design spec. So I guess should be a usage for `react-text` the only thing is that we don't have that tokens implemented there yet